shared/js/background/message-handlers.js

```diff
@@ -98,13 +98,10 @@ export function getTopBlockedByPages (options) {
 }
 
 // Click to load interactions
-export async function initClickToLoad (unused, sender) {
+export async function initClickToLoad (config, sender) {
     await settings.ready()
     const tab = tabManager.get({ tabId: sender.tab.id })
 
-    await tdsStorage.ready('ClickToLoadConfig')
```

Comment:
Can you explain what this is solving?

Reply:
This config is now being passed from the content script (this wasn't actually being used)

Reply:
Ah the other config, thanks for explaining on MM.